crt_portal/cts_forms/migrations/0162_responsetemplate_show_in_dropdown.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.16 on 2023-02-02 20:57
```

Comment:
Can you bundle this with the migration above? It helps to keep a cleaner record of what was migrated with what PR